js/utils/mutation_handlers/cached_handler.js

```diff
@@ -5,8 +5,8 @@ let lastEvent = {
 
 const MUTATION_STABILITY_TIMEOUT = 10; // milliseconds
 
-let isNotCached = timeStamp => title => {
```

Comment:
This was never reassigned, and was only ever called with both arguments

Reply:
See comment by @swapnilraj